src/App.tsx

```diff
@@ -8,7 +7,87 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+const getFilteredTodos = (
+  todos: Todo[], filteredBySelect: string, filteredByQuery: string,
+) => {
+  const todosFiltered = todos.filter(
+    todo => todo.title.includes(filteredByQuery.toLowerCase().trim()),
+  );
+
+  switch (filteredBySelect) {
+    case 'active':
+      return todosFiltered.filter(todo => !todo.completed);
+
+    case 'completed':
+      return todosFiltered.filter(todo => todo.completed);
+
+    case 'all':
+    default:
+      return todosFiltered;
+  }
+};
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredBySelect, setFilteredBySelect] = useState('all');
```

Comment:
one more thing - try to use enum for such predefined values
```suggestion
  const [filteredBySelect, setFilteredBySelect] = useState<SelectedTodos>(SelectedTodos.All);
```